change/@fluentui-react-motion-components-preview-67537ce1-b461-4bec-bbcb-61d9202e6632.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33463&commitId=c74c13b285961c2b4617aa876bf88759c93916ab&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|14| Changed|
|Avatar Converged.Badge Mask RTL.chromium.png|2| Changed|

</details>


  <details><summary><b>Drawer</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay High Contrast.chromium.png|4104| Changed|
|Drawer.Full Overlay Dark Mode.chromium.png|975| Changed|
|Drawer.Full Overlay RTL.chromium.png|1168| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>